web/src/app/pages/profile/profilePage.controller.js

```diff
@@ -113,7 +111,7 @@
 
         getDetails();
 
-        $scope.update = function() {
+        $scope.update = function(profile='/') {
```

Comment:
Its good to initialize the variable outside the function declaration. I hope this is ECMA6 syntax to declare variable value inside the function parameter declaration where all the other code is following ECMA5 style and since this is front-end code it can break in certain browsers

Check the browser compatibility table below Safari, IE and Opera still doesn't support this
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters